custom/conf/app.ini.sample

```diff
@@ -36,6 +36,8 @@ DISABLE_HTTP_GIT = false
 ACCESS_CONTROL_ALLOW_ORIGIN = 
 ; Force ssh:// clone url instead of scp-style uri when default SSH port is used
 USE_COMPAT_SSH_URI = false
+; Close issues as long as a commit on any branch marks it as fixed
+CLOSE_ISSUES_VIA_COMMITS_IN_ANY_BRANCH = false
```

Comment:
This should have prefix `DEFAULT_` like other such options, see https://github.com/go-gitea/gitea/blob/03c8808dfdc075ac2c4b489e2c3a25c7409b84e3/custom/conf/app.ini.sample#L366